src/QsCompiler/Core/Transformations/SyntaxTreeTransformation.fs

```diff
@@ -3,52 +3,37 @@
 
 namespace Microsoft.Quantum.QsCompiler.Transformations.Core
 
+#nowarn "44" // disable deprecation warnings due to the overrides pointing to deprecated methods
+
 open System.Collections.Immutable
 open Microsoft.Quantum.QsCompiler.SyntaxExtensions
 open Microsoft.Quantum.QsCompiler.SyntaxTree
-open Microsoft.Quantum.QsCompiler.Transformations.Core.Utils
 
 
 // setup for syntax tree transformations with internal state
 
 type SyntaxTreeTransformation<'T> private (state: 'T, options: TransformationOptions, _internal_: string) =
-
-    let mutable _Types = new TypeTransformation<'T>(TransformationOptions.Default, _internal_)
-    let mutable _ExpressionKinds = new ExpressionKindTransformation<'T>(TransformationOptions.Default, _internal_)
-    let mutable _Expressions = new ExpressionTransformation<'T>(TransformationOptions.Default, _internal_)
-    let mutable _StatementKinds = new StatementKindTransformation<'T>(TransformationOptions.Default, _internal_)
-    let mutable _Statements = new StatementTransformation<'T>(TransformationOptions.Default, _internal_)
-    let mutable _Namespaces = new NamespaceTransformation<'T>(TransformationOptions.Default, _internal_)
+    inherit CommonTransformationItems()
```

Comment:
Is it intended for users to override these methods only from their class that extends `SyntaxTreeTransformation`, rather than creating a separate class to extend `CommonTransformationItems`?

Reply:
Yes. We could seal it more explicitly, but I believe the internal constructor effectively takes care of that.